pkgs/by-name/sd/sdl2-compat/package.nix

```diff
@@ -48,6 +48,11 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.cmakeBool "SDL2COMPAT_TESTS" finalAttrs.finalPackage.doCheck)
   ];
 
```

Comment:
Assuming there is no harm in setting the environment variable at the build stage, we should just do
```
env.SDL_TESTS_QUICK = 1;
```

Reply:
i just tested, either works perfectly. Your approach is cleaner.